planetaryimage/image.py

```diff
@@ -26,10 +29,25 @@ def open(cls, filename):
         filename : string
             name of file to read as an image file
         """
-        with open(filename, 'rb') as fp:
-            return cls(fp, filename)
+        if filename.endswith('.gz'):
+            try:
+                with gzip.open(filename, 'rb') as fp:
+                    return cls(fp, filename, compression='gz')
+            except AttributeError:
+                raise IOError('GZIP compression not supported until Python 2.6.')
+                return
+        elif filename.endswith('.bz2'):
+            try:
+                with bz2.open(filename, 'rb') as fp:
```

Comment:
Oh and lets not bother with 2.5, 2.6 is still a stretch but I think its still default on RHEL6, which makes it relevant.